lib/java/change-directory/src/main/java/org/enso/change/directory/PosixWorkingDirectory.java

```diff
@@ -0,0 +1,116 @@
+package org.enso.change.directory;
+
+import java.io.File;
+import java.util.List;
+import org.enso.common.Platform;
+import org.graalvm.nativeimage.ImageInfo;
+import org.graalvm.nativeimage.c.CContext;
+import org.graalvm.nativeimage.c.constant.CConstant;
+import org.graalvm.nativeimage.c.function.CFunction;
+import org.graalvm.nativeimage.c.type.CCharPointer;
+import org.graalvm.nativeimage.c.type.CTypeConversion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@CContext(PosixWorkingDirectory.Directives.class)
+public final class PosixWorkingDirectory implements WorkingDirectory {
```

Comment:
This class shouldn't be exported from the `change-directory` module:
```suggestion
final class PosixWorkingDirectory implements WorkingDirectory {
```